gr-blocks/lib/test_tag_variable_rate_ff_impl.cc

```diff
@@ -26,10 +26,12 @@
 
 #include "test_tag_variable_rate_ff_impl.h"
 #include <gnuradio/io_signature.h>
+#include <gnuradio/xoroshiro128p.h>
 #include <string.h>
 #include <iostream>
 #include <iomanip>
 #include <stdexcept>
+#include <cstdint>
```

Comment:
oh, shoot, that one is C++11 or later.

Go for stdint.h, or do an `#if` that checks for C++11 and switch between `cstdint` and `stdint.h`.